packages/client/src/hooks/use-batch-update-charges.ts

```
@@ -0,0 +1,83 @@
+    }
+  }
+`;
+
+type Charges = Extract<
+  BatchUpdateChargesMutation['batchUpdateCharges'],
+  { __typename: 'BatchUpdateChargesSuccessfulResult' }
+>['charges'];
+
+type UseBatchUpdateCharges = {
+  fetching: boolean;
+  batchUpdateCharges: (variables: BatchUpdateChargesMutationVariables) => Promise<Charges | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)